index.html

```diff
@@ -1,299 +1,312 @@
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.
+              </p>
+              <a href="#" class="connect">LET'S CONNECT</a>
+
+              <ul class="media-platform">
+                <li>
+                  <a href="#"
+                    ><img src="./icons/twitter.svg" alt="twitter"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="./icons/Linkedin-icon.svg" alt="Linkedin"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"> <img src="./icons/medium.svg" alt="medium" /></a>
+                </li>
+                <li>
+                  <a href="#"><img src="./icons/github.svg" alt="github" /></a>
+                </li>
+                <li>
+                  <a href="#"><img src="./icons/Vector.svg" alt="Vector" /></a>
+                </li>
+              </ul>
+              <button type="button" class="btn-get">Get my resume</button>
+            </div>
+            <div class="bottom">
+              <ul class="lang-li">
+                <li class="">
+                  <div class="lang-name">
+                    <h2>Languages</h2>
+                    <span
+                      ><img src="./icons/Uniondown.png" alt="down-arrow"
+                    /></span>
+                  </div>
+                  <ul class="classic languages-ul">
+                    <li>
+                      <img src="./photos/Ellipse4.png" alt="js" />
+                      <h3 class="classic-name">Javascript</h3>
+                    </li>
+                    <li>
+                      <img src="./photos/Ellipse1.png" alt="html" />
+                      <h3 class="classic-name">HTML</h3>
+                    </li>
+                    <li>
+                      <img src="./photos/Ellipse2.png" alt="css" />
+                      <h3 class="classic-name">CSS</h3>
+                    </li>
+                  </ul>
+                </li>
+                <li class="common-li">
+                  <div class="lang-name">
+                    <h2>Frameworks</h2>
+                    <span
+                      ><img src="./icons/Uniondown.png" alt="down-arrow"
+                    /></span>
+                  </div>
+                </li>
+                <li class="common-li">
+                  <div class="lang-name">
+                    <h2>Skills</h2>
+                    <span class="skill"
+                      ><img src="./icons/Uniondown.png" alt="down-arrow"
+                    /></span>
+                  </div>
+                </li>
+
+                <li></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </section>
+      
+      <section class="contact-form">

```

Comment:
- [ ] You did an amazing job doing the requested changes from the other reviewer but the footer is still breaking. Kindly fix the width of it for the desktop screen and it will be fixed.

Here you have a screenshot of your design:

![image](https://user-images.githubusercontent.com/74506933/154822098-e12823fc-d7a1-414b-8e69-bf316f544f94.png)

Reply:
Done